contents/docs/ai-engineering/_snippets/anthropic.mdx

```diff
@@ -18,10 +18,12 @@ In the spot where you initialize the Anthropic SDK, import PostHog and our Anthr
 
 ```python 
 from posthog.ai.anthropic import Anthropic
-import posthog
+from posthog import Posthog
 
-posthog.project_api_key = "<ph_project_api_key>"
-posthog.host = "<ph_client_api_host>"
+posthog = Posthog(
+    "<ph_project_api_key>",
+    host="<ph_client_api_host>"
+)
 
 client = Anthropic(
     api_key="sk-ant-api...",  # Replace with your Anthropic API key
```

Comment:
Should we also remove passing the client here? or at least mark it as optional (specifying we use the default if not)

Reply:
*same comment on the others

Reply:
I think I'd lean more towards marking it as optional, since it makes the consumer aware of its existence and capability to customize the options.

Reply:
Talked IRL, we'll keep it as explicitly passed in the docs, while mentioning that it's possible not to pass a client.